src/civic/index.js

```diff
@@ -117,18 +117,31 @@ const validateEvidenceSpec = ajv.compile({
 const translateRelevance = (evidenceType, evidenceDirection, clinicalSignificance) => {
     switch (evidenceType) { // eslint-disable-line default-case
         case 'Predictive': {
-            switch (clinicalSignificance) { // eslint-disable-line default-case
-                case 'Sensitivity':
-                case 'Adverse Response':
-                case 'Reduced Sensitivity':
+            if (evidenceDirection === 'Does Not Support') {
+                switch (clinicalSignificance) { // eslint-disable-line default-case
+                    case 'Sensitivity':
 
-                case 'Resistance': {
-                    return clinicalSignificance.toLowerCase();
+                    case 'Sensitivity/Response': {
+                        return 'no response';
+                    }
                 }
+            } else if (evidenceDirection === 'Supports') {
+                switch (clinicalSignificance) { // eslint-disable-line default-case
+                    case 'Sensitivity':
+                    case 'Adverse Response':
+                    case 'Reduced Sensitivity':
+
+                    case 'Resistance': {
+                        return clinicalSignificance.toLowerCase();
+                    }
 
-                case 'Sensitivity/Response': { return 'sensitivity'; }
+                    case 'Sensitivity/Response': { return 'sensitivity'; }
+                }
             }
-            break;
+
+            throw new Error(
```

Comment:
you don't need this error. It is unreachable because it is after the break statement. When the code breaks out of the switch it will raise the error at the end anyway

Reply:
I've pushed changes to replace the Error with a break.